data/build-flavors/generic.toml

```diff
@@ -7,6 +7,7 @@ packages = [
   # QEMU and Xen guest tools exist for multiple architectures
   "qemu-guest-agent",
   "vyos-xe-guest-utilities",
+  "ipmitool"
```

Comment:
If there is no CLI for the package, it should be in the `vyos-user-utils` https://github.com/vyos/vyos-user-utils/blob/current/debian/control

Reply:
Thanks, I will create a PR there.